src/structures/TextChannel.js

```diff
@@ -73,11 +80,29 @@ class TextChannel extends GuildChannel {
       this.lastPinTimestamp = data.last_pin_timestamp ? new Date(data.last_pin_timestamp).getTime() : null;
     }
 
+    if ('default_auto_archive_duration' in data) {
+      /**
+       * The default auto archive duration for newly created threads in this channel
+       * @type {ThreadAutoArchiveDuration?}
```

Comment:
Isn't this technically nullable or possibly undefined

Reply:
? at the end usually means undefined, looks like the site renders it as nullable though. definitely possible to be undefined, but its not nullable I think

Reply:
I think our docgen only supports `?Type` properly, and `Type?` yields the same because of this.

But I would argue doing `?Type` is reasonable here, even if its not null, indicating nullish?